diff --git a/src/inprint2.c b/src/inprint2.c
index 45e3ce5..ee2cec8 100644
--- a/src/inprint2.c
+++ b/src/inprint2.c
@@ -96,6 +96,7 @@ void inprint(SDL_Renderer *dst, const char *str, Uint32 x, Uint32 y, const Uint3
 
   for (; *str; str++) {
     int id = (int)*str - font_offset;
+    if (id == -1) id = 95;
 #if (CHARACTERS_PER_COLUMN != 1)
     int row = id / CHARACTERS_PER_ROW;
     int col = id % CHARACTERS_PER_ROW;
